components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -129,6 +135,12 @@ void AIChatTabHelper::RemoveObserver(Observer* observer) {
 }
 
 void AIChatTabHelper::MaybeGeneratePageText() {
+  const GURL url = web_contents()->GetURL();
```

Comment:
Please don't use `WebContents::GetURL`

```
  // Gets the URL that is currently being displayed, if there is one.
  // This method is deprecated. DO NOT USE! [...]
```

For this case, `GetLastCommittedURL` seems appropriate.

Reply:
changed